Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -73,7 +73,7 @@ + (NSString *)getURIScheme {
 }
 
 + (NSString *)getAppVersion {
-    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleVersion"];
+    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
```

Comment:
Huh, that is a fair point. I don't think anything is enforced with the Build Number except that it be different than the previous.